### PR TITLE
#8229: Report child functions durations per op in op report

### PR DESCRIPTION
**Pull request by mo-tenstorrent**
This PR brings adding child function durations such `CompileProgram` to TTNN ops row in op report.

Sample op report:

Last columns in the report, 
BERT: 
[ops_perf_results_2024_05_27_13_45_33.csv](https://github.com/tenstorrent/tt-metal/files/15457102/ops_perf_results_2024_05_27_13_45_33.csv)

### CODE REVIEW

**Comment by TT-billteng** [comment]
this requires a bump to tracy submodule?

**Comment by mo-tenstorrent** [comment]
> this requires a bump to tracy submodule?

It does, it requires tracy's `csvexport` to dump all the child classes that have a TTDNN op as parent.

File: tt_eager/tracy.py [hunk 1]
```diff
@@ -29,6 +29,8 @@
 
 import tracy_state
 
+DEFAULT_CHILD_CALLS = ["CompileProgram", "HWCommandQueue_write_buffer"]
```
Comment (mo-tenstorrent) [inline tt_eager/tracy.py:32]: @davorchap @cglagovichTT We can ask for child calls in command line but this line here is a list of functions that will always be included for the op if it is present. What are the child functions you are interested in that should be included by default?

**Comment by mo-tenstorrent** [comment]
Device perf only failing on model regression on SD, profiler infra has no issues: https://github.com/tenstorrent/tt-metal/actions/runs/9230242418

T3K profiler passing: https://github.com/tenstorrent/tt-metal/actions/runs/9255999380

Post commit passing : https://github.com/tenstorrent/tt-metal/actions/runs/9255996440

**Review by tt-rkim — APPROVED** [review]

**Pull request merged**